### ISSUE TITLE
Terraform Deployment Failing for 0.7

### ISSUE THREAD

**Issue by dawsonpa** [google/exposure-notifications-verification-server#525]
### TL;DR
Terraform Deployment is not working on new GCP Project for 0.7

**Expected behavior**
Successfully deploy Verification Server via Terraform.

**Observed behavior**
Terraform Deployment Fails
### Reproduction
1.  Followed the Terraform Deployment README here https://github.com/google/exposure-notifications-verification-server/tree/main/terraform. Used the Quick Setup and Local Development/Testing configuration

**Screenshots**

<img width="1650" alt="Screen Shot 2020-09-11 at 10 25 59 AM" src="https://user-images.githubusercontent.com/7635070/92937695-abafb400-f419-11ea-8207-cd92753553a5.png">

**Environment**

- OS: Mac 
- Version: 10.15.5 

**Additional information**
Terraform Version - v0.13.2
I am also using the local development  configuration based on the README for terraform deployments
<img width="431" alt="Screen Shot 2020-09-11 at 10 25 01 AM" src="https://user-images.githubusercontent.com/7635070/92937237-14e2f780-f419-11ea-981c-1df95b1eeb85.png">

**Comment by sethvargo** [comment]
/assign

Can you share the full Terraform output please? There's probably an error above with something failed.

**Comment by sethvargo** [comment]
And what do the logs for that service say? You can find them in the Google Cloud console.

**Comment by dawsonpa** [comment]
> Can you share the full Terraform output please? There's probably an error above with something failed.

@sethvargo The full terraform output can be found here.

https://gist.github.com/dawsonpa/612739dbf62c48dc78ecdebdfc481213

**Comment by dawsonpa** [comment]
@sethvargo I am seeing an error in the Google Cloud Console for that service:

![image](https://user-images.githubusercontent.com/7635070/92943861-495ab180-f421-11ea-8217-979d1c63967e.png)

**Comment by dawsonpa** [comment]
@sethvargo I was able to get it to work by adding the following in the service_server.tf file

![image](https://user-images.githubusercontent.com/7635070/92945502-5a0c2700-f423-11ea-9fa6-8abede5bfaf8.png)

**Comment by sethvargo** [comment]
Hi @dawsonpa - this is expected and is [documented here](https://github.com/google/exposure-notifications-verification-server/blob/main/terraform/README.md#production-setup).

/close
